Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,263 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+    var description = String()
+    var timestamp = String()
+}
+
+enum Stage {
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+}
+
+// MARK: - Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
+    static func saveToKeychain(loginInfo: LoginInfo) {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = loginInfo.netid as AnyObject
+        keychain["password"] = loginInfo.password as AnyObject
+    }
+    
+    static func removeKeychainLoginInfo() {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = nil
+        keychain["password"] = nil
+    }
+    
+    static func loadFromKeychain() -> LoginInfo? {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        guard let netid = keychain["netid"] as? String, let password = keychain["password"] as? String else {
+            return nil
+        }
+        return (netid: netid, password: password)
+    }
+    
+}
+
+// MARK: - Connection Handler
+
+private protocol BRBConnectionHandlerDelegate {
+
```

Comment:
Do we add these spaces for protocol declarations? I think they look bad.

Reply:
I did not have the spacing before, but Kevin told me to add it

Reply:
Oh oops nvm then :)